flower_shop/security/ir_rule.xml

```diff
@@ -0,0 +1,18 @@
+<odoo>
+    <data noupdate="1">
+
+        <record id="global_access_rule" model="ir.rule">
+            <field name="name">Global Product Access</field>
+            <field name="model_id" ref="model_product_product"/>
+            <field name="domain_force">[('is_flower', '=', False)]</field>
```

Comment:
```suggestion
            <field name="domain_force">[(1,'=',1)] if user.user_has_groups('flower_shop.group_gardener') else [('is_flower','=',False)]</field>
```

Reply:
I already used that command but didn't work, shows error..Idk why